style.css

```diff
@@ -60,7 +60,7 @@ nav {
 
 .hero {
   background-position: fixed;
-  background-image: url("/Pictures/headershapes.png");
+  background-image: url("./Pictures/headershapes.png");
   gap: 12px;
```

Comment:
- [ ] Kindly make sure your page doesn't overlap on the right. the width must be `100%` and not more.
your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193893517-8314d6e2-7691-4336-b58e-cb8dd1b315e0.png)
![image](https://user-images.githubusercontent.com/97100091/193893735-25319f27-736c-4378-b88a-7408353e444a.png)

Reply:
@christianbiring1 Thank you for pointing out this issue🤗. I have adjusted the width to 100% now.